graphs/README.md

```diff
@@ -3,7 +3,6 @@ We have included a set of dashboards for Grafana in this demo application.
 Most of the source of these dashboards look exactly the same. For the sake of
 keeping our code DRY, we have decided to generate our dashboards using [grafanalib](https://github.com/weaveworks/grafanalib).
 
-
 # Requirements
 If you don't have Python installed on your computer but do have Docker, then Docker and Make will
```

Comment:
Not sure if we even have to talk about this, just default to docker. Also, I cannot see a `Makefile` here.